app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
+                    ),
+                    Presentation::ReviewStatusReceived => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusReceived,
+                        "p.status = 'Received' AND e.published = 0 AND
+                            selp IS NOT NULL AND ( 
+                                (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                                selp.submission_begin_date <= '$now_utc' AND 
+                                selp.submission_end_date >= '$now_utc' AND
+                                selp.selection_begin_date <= '$now_utc' AND 
+                                selp.selection_end_date >= '$now_utc'
+                            )"
+                    ),
+                    Presentation::ReviewStatusInReview => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusInReview,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND
+                            selp IS NOT null AND (
+                            (selp.submission_lock_down_presentation_status_date IS NOT NULL AND selp.submission_lock_down_presentation_status_date > '$now_utc') OR 
+                            (
+                                (selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+                                AND 
+                                (selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+                            )
+                         )"
+                    ),
+                    Presentation::ReviewStatusPublished => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusPublished,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
+                            selp IS NOT null AND ( 
+                            (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                            (
+                                (selp.submission_begin_date <= '$now_utc' AND selp.submission_end_date >= '$now_utc') 
+                                OR selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc'
+                            )
+                        )"
+                    ),
+                    Presentation::ReviewStatusAccepted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusAccepted,
+                        sprintf('(p.status = \'Received\' OR p.status = \'Accepted\') AND e.published = 0 AND 
```

Comment:
this e.published  = 0 is not correct ( review accepted status ) https://github.com/OpenStackweb/summit-api/blob/2d46fc910d303115fc680f5dc9763acbf68728bf/app/Repositories/Summit/DoctrineSummitEventRepository.php#L378